### ISSUE TITLE
Question about Filename Formatting

### ISSUE THREAD

**Issue by Latentnaut** [giriss/comfy-image-saver#19]
Is it possible to use built-in feature to formatting the filename or foldername? Like:

%node_name.widget_name%

https://blenderneko.github.io/ComfyUI-docs/Interface/SaveFileFormatting/#search-and-replace-strings

**Comment by Jellybit** [comment]
Yes please. When doing tests, I love to put the name of the lora in the file name, sometimes the seed, sometimes the checkpoint, or reference an iteration number, etc... All my file names are set up so that I can type a project name in a box, and the generic image saver makes a folder for that by looking at what is typed in that box. I use this stuff constantly.
